assets/js/lib/test-utils/factories/advisoryErrata.js

```diff
@@ -2,6 +2,14 @@ import { faker } from '@faker-js/faker';
 import { Factory } from 'fishery';
 import { advisoryType } from './relevantPatches';
 
+const affectedPackageFactory = Factory.define(() => ({
+  name: faker.animal.cat().toLowerCase(),
```

Comment:
```suggestion
const affectedPackageFactory = Factory.define(({ sequence }) => ({
  name: `${faker.animal.cat().toLowerCase()}${sequence}`,
```

Using sequences is sort of mandatory if you build lists out of them and want to use `screen.getByText`, even with `{ exact: false }`, otherwise we risk duplicated generated texts and flaky tests :+1: